<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>W_tabela</class>
 <widget class="QMainWindow" name="W_tabela">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="table_list">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>1</y>
      <width>481</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="add_t">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>380</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Adicionar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_t">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>380</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Apagar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_t">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>350</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>490</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionSalvar_no_Banco_de_Dados"/>
   </widget>
   <addaction name="menuArquivo"/>
  </widget>
  <action name="actionSalvar_no_Banco_de_Dados">
   <property name="text">
    <string>Salvar no Banco de Dados</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
